gulpfile.babel.js

```diff
@@ -77,3 +77,12 @@ gulp.task('stop', finish => {
     }, 1000);
   });
 });
+
+
+gulp.task('platform-test', function (cb) {
+  child_process.exec('sh ./bootstrap/test/platform_test.sh', function (err, stdout, stderr) {
```

Comment:
@romandev, would you explain again about implementing platform_test in javascript? :-)